### PR TITLE
Add missing call on items

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: pyramid_closure/views.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 def pairwise(iterable):
     if isinstance(iterable, dict):
-        return iterable
+        return iterable.items()
```
Comment (elemoine) [inline pyramid_closure/views.py]: `izip` returns an iterator, so, for consistency, I'd rather use `iteritems()` here. I know Python 3 doesn't have `iteritems`, but we should use `six` if we want to be compatible with both Python 2 and 3.

**Comment by elemoine** [comment]
If you agree please use `iteritems` and merge. Thanks.

**Comment by sbrunner** [comment]
Thanks for the review :-)

**Comment by sbrunner** [comment]
@elemoine can you create a new egg with this change ?

**Comment by elemoine** [comment]
We do not currently create eggs for pyramid_closure. pyramid_closure is downloaded from GitHub. See https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/requirements.txt for example.

**Comment by sbrunner** [comment]
Oups sorry, thanks :-)

**Pull request merged**